frontend/public/components/feedback-local.js

```diff
@@ -0,0 +1,58 @@
+import i18n from 'i18next';
+
+export const Feedbacki18n = {
+  getSupport: i18n.t('public~Get help from Red Hat support.'),
```

Comment:
These strings won't get updated when the language changes in console (i.e. when the user changes their language preference from the user settings UI).

Reply:
To fix, you can use the `t` from `useTranslation` hook in a component.